src/test/java/htsjdk/variant/variantcontext/VariantContextUnitTest.java

```diff
@@ -108,109 +108,190 @@ public void testDetermineTypes() {
 
         // test REF
         List<Allele> alleles = Arrays.asList(Tref);
```

Comment:
Oh, I figured this could be done programmatically with a dataprovider instead of needing quite so much boilerplate.  Sorry for so much typing!

Reply:
Ah, yeah, we do.  This test didn't for some reason, so I can see why it's confusing.